### PR TITLE
Complete Layers panel context menu description

### LINKED ISSUE #696: [FEATURE] Add a show/hide all context menu for layer tree symbol items
Original commit: https://github.com/qgis/QGIS/commit/91f7918e21e5251a149515200b0f566f8f02c85b by nyalldawson

Allows toggling on/off all the symbol items for categorized/graduated/
rule based layers via the right click menu on an item. Previously
you'd have to manually toggle each item one-by-one.

Fix #13458

### LINKED ISSUE #706: [FEATURE] Edit legend symbols directly from layer tree
Original commit: https://github.com/qgis/QGIS/commit/6e140b97fddbc7c81ebc56f06cd4ef54c9f82e75 by nyalldawson

This adds a new "edit symbol" item to the right-click menu for a
renderer child legend item (eg categories for the categorised
renderer). Selecting it opens a symbol editor dialog which allows
for directly editing the classes symbol. It's much faster than
opening the layer properties and going through the style tab.

### LINKED ISSUE #708: [FEATURE] Directly set class symbol color from context menu in legend
Original commit: https://github.com/qgis/QGIS/commit/2026fce135d379605b0d5b2777dd20ebbf852707 by nyalldawson

Shows a color wheel widget in the menu, which allows you to
interactively edit the color for a classes' symbol.

### LINKED ISSUE #709: [FEATURE] Double clicking a legend node opens symbol editor directly
Original commit: https://github.com/qgis/QGIS/commit/c7c5c007be90076ba023090152410e2294bd402b by nyalldawson

Unfortunately this lazy coder did not write a description... :-(

### LINKED ISSUE #744: [FEATURE] Add color wheel and edit symbol shortcuts to style menu
Original commit: https://github.com/qgis/QGIS/commit/974cc35759bc46a84c2d462fa0faf8cfbe079528 by nyalldawson

...for vector layers with single symbol renderer. This completes
earlier work where symbols from categorized/graduated/rule based
renderers can be quickly changed through the legend item context
menu.

### LINKED ISSUE #745: [FEATURE] Show recent colors in layer right click menus
Original commit: https://github.com/qgis/QGIS/commit/38baaafd0539d0031d9e1e3990c77a0103d137e2 by nyalldawson

Unfortunately this lazy coder did not write a description... :-(

### PR DESCRIPTION
**Pull request by DelazJ**
Fix #696, Fix #706, Fix #708, Fix #709, Fix #744, Fix #745

Complete list of options available in context menu
Add description of Styles sub-items

### CODE REVIEW

File: source/docs/user_manual/working_with_vector/vector_properties.rst [hunk 1]
```diff
@@ -1965,9 +1970,6 @@ load and manage styles.
 A style stores any information set in the layer properties dialog to render
 or interact with the features (including symbology, labeling, action, diagram...
 settings).
-Styles can be stored inside the project, in a file (``.qml`` or ``.sld``) or
```
Comment (yjacolin) [inline source/docs/user_manual/working_with_vector/vector_properties.rst]: why did you remove this part?

**Comment by yjacolin** [comment]
LGTM

File: source/docs/user_manual/working_with_vector/vector_properties.rst (hunk 1 above)
Comment (DelazJ) [inline source/docs/user_manual/working_with_vector/vector_properties.rst]: because of https://github.com/DelazJ/QGIS-Documentation/blob/ecb04fbc15da9b9c2b59eca8fc24913ca226ef83/source/docs/user_manual/working_with_vector/vector_properties.rst#storing-style-in-a-file-or-a-database
